s2spy/rgdr/rgdr.py

```diff
@@ -57,12 +57,7 @@ def cluster_area(ds: XrType, cluster_label: float) -> float:
     Returns:
         float: Area of the cluster `cluster_label`.
     """
-    return (
-        ds["area"]
-        .where(ds["cluster_labels"] == cluster_label)
-        .sum(skipna=True)
-        .values.item()
-    )
+    return ds["area"].where(ds["cluster_labels"] == cluster_label).sum(skipna=True).values.item()
```

Comment:
Something went wrong here, Yang. These lines are >100 long. See my suggestion for the pyproject.toml file

Reply:
Solved!